src/main/java/teammates/logic/core/TaskQueuesLogic.java

```diff
@@ -35,13 +36,18 @@ public void addDeferredTask(TaskWrapper task, long countdownTime) {
             taskToBeAdded.countdownMillis(countdownTime);
         }
 
-        for (Map.Entry<String, String[]> entry : task.getParamMap().entrySet()) {
-            String name = entry.getKey();
-            String[] values = entry.getValue();
+        // Task queue API only allows either parameter map or body, not both
```

Comment:
If we allow both parameters and body, shouldn't that be more consistent with the REST API

Reply:
No, that's not the issue. By "task queue API" I am referring to GAE's API. If they allow both I would have allowed both.